docs/source/user/settings.rst

```diff
@@ -150,7 +150,7 @@ the following to your project ``settings.py``:
 
 **Default**: ``[]``
 
-List of host IP addresses allowed to consume the freeradius
+List of host IP addresses or subnet allowed to consume the freeradius
```

Comment:
I think it should be `subnets` since multiple items can be allowed in this setting.